.github/workflows/add-files-changed-label.yml

```
@@ -0,0 +1,165 @@
+          REPO_OWNER: ${{ github.repository_owner }}
+          REPO_NAME: ${{ github.event.repository.name }}
+        run: |
+          echo "Starting Files Changed Label workflow for PR #$PR_NUMBER in $REPO_OWNER/$REPO_NAME"
+          
+          # Get the number of files changed in the PR
+          echo "Fetching files changed information..."
+          FILES_API_RESPONSE=$(curl -s -X GET \
+            -H "Authorization: token $GITHUB_TOKEN" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/pulls/$PR_NUMBER/files")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)